src/rest_api/main.clj

```diff
@@ -0,0 +1,33 @@
+(ns rest-api.main
+  (:require
+   [compojure.api.sweet :as sweet]
+   [mount.core :as mount]
+   [rest-api.classes.gene :as gene]
+   [rest-api.classes.transcript :as transcript]))
+
+(defn init []
+  (mount/start))
+
+(def ^:private all-routes
+  [gene/routes
+   transcript/routes])
+
+;; `validatorUrl` doesn't work with private urls, see:
+;; https://github.com/Orange-OpenSource/angular-swagger-ui/issues/43
+(def app
```

Comment:
The main ring handler entry point can now be used with a plain `def` or the `defapi` macro from compojure-api.
This is because the call for `datomic.api/db` has been deferred to when the request is handled, as opposed to up-front in the top-level ring handler.